src/background.xml

```diff
@@ -5,66 +5,76 @@
       Transport Capacity measurement was known to be well beyond
       our capabilities. Even at the time that Framework for IP
       Performance Metrics 
-      <xref target="RFC3148" /> was written we knew that we didn't
+      <xref target="RFC2330" /> was written we knew that we didn't
       fully understand the problem. Now, by hindsight we understand
-      why BTC is such a hard problem:
+      why assessing BTC is such a hard problem:
       <list style="symbols">
 
         <t>TCP is a control system with circular dependencies -
         everything affects performance, including components that
-        are explicitly not part of the test.</t>
+        are explicitly not part of the test (for example, the host 
+        processing power is not in-scope of path performance tests).</t>
 
-        <t>Congestion control is an equilibrium process, such that
-        transport protocols change the packet transfer statistics
```

Comment:
I meant "equilibrium process" as in https://en.wikipedia.org/wiki/Chemical_equilibrium.    "seeking equilibrium" to most people means asymptotically converging to zero slope or a fixed value, which is true but not the point I was trying to make.